packages/tools/client-logger/app-insights-logger/src/test/ClientUtilities.ts

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ConnectionState } from "@fluidframework/container-loader";
+import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import {
+	TinyliciousClient,
+	TinyliciousContainerServices,
+} from "@fluidframework/tinylicious-client";
+// import { AppInsightsLogger } from "../appInsightsLogger";
+
+/**
+ * This module contains Fluid Client utilities, including Container creation / loading.
+ */
+
+/**
+ * Type returned from when creating / loading the Container.
+ */
+export interface ContainerLoadResult {
+	container: IFluidContainer;
+	services: TinyliciousContainerServices;
+}
+
+/**
+ * Basic information about the container, as well as the associated audience.
+ */
+export interface ContainerInfo {
+	/**
+	 * {@link ContainerInfo.container}'s unique ID.
+	 */
+	containerId: string;
+
+	/**
+	 * The initialized Fluid Container.
+	 */
+	container: IFluidContainer;
+
+	/**
+	 * Optional nickname for the Container to be used in the debugger and associated UI.
+	 */
+	containerNickname?: string;
+}
+
+function initializeTinyliciousClient(): TinyliciousClient {
+	console.log(`Initializing Tinylicious client on port ${process.env.PORT}...`);
+	return new TinyliciousClient({
+		// logger: new AppInsightsLogger({
```

Comment:
You can't instantiate the logger without an App Insights configuration that is pointing to a live instance. The idea is if someone wanted to use this test, they'd just have to uncomment this portion and add their own configuration

Reply:
I'm actually going to adjust this to not be commented out after further thought. It will make the test break and output a stack trace to update the string in the browser console which is more straight forward.